apps/graphql/src/apps/itinerary/helpers/Itineraries.js

```diff
@@ -36,6 +41,17 @@ export const mapVehicle = (type: ?string, uniqueNo: ?string) => ({
   uniqueNo: uniqueNo ?? null,
 });
 
+export const slugsNeedParsing = (location: ?SearchLocation): boolean => {
+  return location?.ids == null && location?.slugs != null;
+};
+
+export const unmaskReplaceIds = (location: ?SearchLocation): SearchLocation => {
+  return {
+    ...location,
+    ...(location && location.ids && { ids: unmaskID(location.ids) }),
```

Comment:
You could also use nullish coalescing and optional chaining here

Reply:
Done! Usually when I am fiddling with those I get some Flow infer errors, still have to get used to that.

Reply:
Yeah, we are early adopters, so there still might be some things that will improve. 

😄